messenger.rst

```diff
@@ -177,50 +381,104 @@ like this:
 The first argument is the receiver's service name. It might have been created by
 your ``transports`` configuration or it can be your own receiver.
 
-Multiple buses
+Multiple Buses
 --------------
 
-If you are interested into architectures like CQRS, you might want to have multiple
+If you are interested in architectures like CQRS, you might want to have multiple
 buses within your application.
 
-You can create multiple buses (in this example, a command and an event bus) like
+You can create multiple buses (in this example, a command bus and an event bus) like
 this:
 
-.. code-block:: yaml
-
-    framework:
-        messenger:
-            # The bus that is going to be injected when injecting MessageBusInterface:
-            default_bus: commands
-
-            # Create buses
-            buses:
-                messenger.bus.commands: ~
-                messenger.bus.events: ~
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/messenger.yaml
+        framework:
+            messenger:
+                # The bus that is going to be injected when injecting MessageBusInterface:
+                default_bus: commands
+
+                # Create buses
+                buses:
+                    messenger.bus.commands: ~
+                    messenger.bus.events: ~
+
+    .. code-block:: xml
+
+        <!-- config/packages/messenger.xml -->
+        <container xmlns="http://symfony.com/schema/dic/symfony"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:messenger default-bus="commands">
```

Comment:
```diff
- <framework:messenger default-bus="commands">
+ <framework:messenger default-bus="messenger.bus.commands">
```

(same in other formats)

Perhaps we could recommend using something more simple like `command_bus`, `query_bus` and `event_bus` rather than long prefixed ids?